docs/adr/0021-move-verification-tests.md

```diff
@@ -0,0 +1,25 @@
+# 1. Avoid listing file counts unless user is authenticated
+
+Date: 2024-08-12
+
+## Status
+
+Accepted
+
+## Context
+
+Regular CI runs (on push, PR etc.) ran all tests, including the verification tests.
+
+These verification tests made requests to the GitHub API.
+
+Lots of CI runs in close succession could result in this API returning 403 errors (see #4434), causing test failures.
+
+## Decision
+
+Verification tests will be moved to their own GitHub workflow, which will be run on its own nightly schedule.
+
+## Consequences
+
```

Comment:
Another (good) consequence is that our deployment pipeline is no longer dependent on the availability of the GitHub or Open Codelists APIs.